### PR TITLE
Add PHP Code Standards Fixer configuration

### PR DESCRIPTION
**Pull request by jeromegamez**
This is a small one, but it would help contributors making sure that the coding standards are met.

Cheers
:octocat: Jérôme

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+return Symfony\CS\Config\Config::create()
+    ->finder(
+        Symfony\CS\Finder\DefaultFinder::create()
+            ->exclude('vendor')
+            ->in(__DIR__)
+    )
+;
```
Comment (GrahamCampbell) [inline .php_cs]: Missing  new line at eof.

**Comment by GrahamCampbell** [comment]
You need to specify the level since the default is different between php-cs-fixer 1.x and 2.x.

**Comment by egeloen** [comment]
:+1: Can you add the level in order to get the same behavior between both versions of the php cs fixer as explained by @GrahamCampbell ? Thanks!

**Comment by jeromegamez** [comment]
Added new line and the level specification as suggested, thanks @GrahamCampbell!

**Comment by egeloen** [comment]
Thank you!

**Pull request merged**